govcd/edgegateway.go

```diff
@@ -200,82 +200,82 @@ func (eGW *EdgeGateway) RemoveNATPortRule(networkHref, natType, externalIP, exte
 
 }
 
-// Temporary fix to support parsing network Id from HREF
-// TODO: remove this function when ruleId is available
-func extractObjectIDfromPath(locationPath string) (string, error) {
-	if locationPath == "" {
-		return "", fmt.Errorf("unable to get ID from empty path")
+// RemoveNATRule removes NAT rule and handles task. Return errors when issue risen.
```

Comment:
For clarity: "removes NAT rule and handles task" -> "removes NAT rule identified by ID and handles task"

Grammar: "Return errors when issue risen" -> "Returns error if issues rise"

Reply:
changed